js/index.js

```diff
@@ -0,0 +1,19 @@
+$(document).ready(function(){
+
```

Comment:
@SOyeniyi801 You missed adding the function to turn the bulbs black. We can talk about this more in class.
-you can create a function similar to the ones you have but target the "bulbs" to turn black.